spec/models/landing_page/block/hero_spec.rb

```diff
@@ -3,14 +3,62 @@
     { "type" => "hero",
       "image" => {
         "alt" => "some alt text",
-        "sources" => {
-          "desktop" => "landing_page/desktop.jpeg",
-          "desktop_2x" => "landing_page/desktop_2x.jpeg",
-          "mobile" => "landing_page/mobile.jpeg",
-          "mobile_2x" => "landing_page/mobile_2x.jpeg",
-          "tablet" => "landing_page/tablet.jpeg",
-          "tablet_2x" => "landing_page/tablet_2x.jpeg",
-        },
+        "sources" => [
```

Comment:
I could probably do a much better job of structuring this payload on the whitehall side. All the way down to this if we wanted:

```
      "image" => {
        "alt" => "some alt text",
        "sources" => {
          "hero_desktop_1x" => "landing_page/desktop.jpeg",
          "hero_desktop_2x" => "landing_page/desktop_2x.jpeg",
          "hero_tablet_1x" => "landing_page/tablet.jpeg",
          "hero_tablet_2x" => "landing_page/tablet_2x.jpeg",
          "hero_mobile_1x" => "landing_page/mobile.jpeg",
          "hero_mobile_2x" => "landing_page/mobile_2x.jpeg",
        },
```

... and whitehall could validate that the alt text is nil or the same for all of the images etc.

The other metadata (created_at, caption etc.) is always going to be a bit confusing while we're uploading three separate images (mobile, tablet, desktop) with 2 variants each.

https://github.com/alphagov/whitehall/pull/9579 does produce the payload in the current format - I haven't attempted to improve it yet. I'm happy to stick with the current format if you like, but I'd probably vote to improve it before the two PRs merge.

Reply:
Okay, let's simplify it. I was hoping that maybe leaving the code the way it is would give us some more general support in the future, but I'm forgetting YAGNI. Let's make the code simpler now, and we can complicate it later if we need to.

Reply:
I'll confirm what the structure will be once I've updated my second PR

Reply:
Current proposed structure:

https://github.com/alphagov/whitehall/pull/9579/files#diff-bec4c63fd7b93e9d9e92a09e0efaaa528c3c3ee2c1a09e5b689970a3d870b1f8R158-R192

Reply:
Hmm, are we losing the alt text entirely?

Reply:
I've updated, but kept the top-level alt.